ooniprobe/Utility/VersionUtility.mm

```diff
@@ -0,0 +1,16 @@
+//
+//  VersionUtility.m
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 24/09/17.
+//  Copyright © 2017 Simone Basso. All rights reserved.
```

Comment:
This should be the standard MK header

Reply:
Or just not be there as for the rest of the code.

Reply:
It should probably mention `OONI Probe` not `MK`. TBH, I am also fine it removing all headers if that's what you like the most :-)